### ISSUE TITLE
Catch broken coastlines with semicolon separated values

### ISSUE THREAD

**Issue by b1tw153** [DavidKarlas/OsmNightWatch#15]
Include ways with semicolon separated values of natural=* where one of the values is "coastline" in the list of broken coastlines. The `natural=coastline;bare_rock` tag is the most common example of this.

Just a suggestion for an added feature, since these ways are also broken coastlines.

**Comment by DavidKarlas** [comment]
Problem is that most if not all software working with coastlines only handles exactly natural=coastline and not looking for mult-value tag separated with  `;`, https://github.com/osmcode/osmcoastline is such example as far as I understand...
If osmcoastline software and few others would start handing `;` or at least to be shown as landmass on osm.org I would be fine with this change logic in NighWatch...
See https://overpass-turbo.eu/s/1DfE for example they are not rendered as islands on OSM today...

**Comment by b1tw153** [comment]
Thanks for looking at the issue. I'm not sure from your answer -- did you decide to flag values like `natural=coastline;bare_rock` as broken coastlines?

**Comment by DavidKarlas** [comment]
Oh no, I totally misunderstood your ask, I thought you wanted me to process anything with `coastline;*` as valid coastline, but instead you are asking me to flag those as problems?

**Comment by b1tw153** [comment]
Yes, that's right -- flag `natural=coastline;*` as problems, for all the reasons you mentioned.

Sorry if my original post wasn't clear!

**Comment by DavidKarlas** [comment]
How would I fix natural=coastline;bare_rock? Duplicate existing way and put tags on existing way natural=coastline and on duplicated geometry I would put natural=bare_rock?

**Comment by b1tw153** [comment]
Yes, exactly. That's one way of separating the tags.

It’s also possible to use a multipolygon with natural=bare_rock and with the coastline way as a member. That’s what the Wiki suggests (https://wiki.openstreetmap.org/wiki/Tag:natural%3Dcoastline#How_to_enter_the_data) but I believe the preferred approach is to duplicate the geometry.
